multi_subject/process_data.sh

```diff
@@ -80,11 +80,11 @@ segment_if_does_not_exist() {
   if [[ -e "${FILESEGMANUAL}" ]]; then
     echo "Found! Using manual segmentation."
     rsync -avzh "${FILESEGMANUAL}" "${FILESEG}".nii.gz
-    sct_qc -i "${file}".nii.gz -s "${FILESEG}".nii.gz -p sct_deepseg_sc -qc "${PATH_QC}" -qc-subject "${SUBJECT}"
+    sct_qc -i "${file}".nii.gz -s "${FILESEG}".nii.gz -p sct_deepseg -qc "${PATH_QC}" -qc-subject "${SUBJECT}"
```

Comment:
weird, this does not appear in our doc:

~~~
julien-macbook:~/sct_course_data/single_subject/data/t1 $ sct_qc 

--
Spinal Cord Toolbox (git-master-f226d04768b64d5c68e00a442f209de1eea153cd)

sct_qc 
--

usage: sct_qc -i IMAGE -p
              {sct_propseg,sct_deepseg_sc,sct_deepseg_gm,sct_deepseg_lesion,sct_register_multimodal,sct_register_to_template,sct_warp_template,sct_label_vertebrae,sct_detect_pmj,sct_label_utils,sct_get_centerline,sct_fmri_moco,sct_dmri_moco,sct_image_stitch,sct_fmri_compute_tsnr}
~~~

Reply:
Ah shoot, I assumed it was there but it seems like it isn't. Thanks for catching this.

Reply:
i tested my script-- it works-- let's go with my version